tests/sass/function-name-format.scss

```diff
@@ -0,0 +1,17 @@
+$foo: kabab-case();
+$foo: snake_case();
+$foo: camelCase();
+$foo: PascalCase();
+$foo: Camel_Snake_Case();
+$foo: SCREAMING_SNAKE_CASE();
+$foo: _with-leading-underscore();
+$foo: _does_NOT-fitSTANDARD();
+
+.class {
+  content: snake_case();
+}
+
+@function camelCase($x, $y) {
```

Comment:
By check, do you mean run the test file through Sass and verify it can be processed correctly?